indico/modules/events/registration/models/items.py

```diff
@@ -76,20 +106,25 @@ class RegistrationFormItem(db.Model):
         )
     )
 
-    # The parent of the item and the children backref
-    parent = db.relationship(
+    # The children of the item and the parent backref
+    children = db.relationship(
         'RegistrationFormItem',
         lazy=True,
         backref=db.backref(
```

Comment:
Maybe this would be a good moment to drop using `backref` and start adding the `relationship` in both places? @ThiefMaster